sbt-cross/src/main/scala/sbtcross/SbtCrossPlugin.scala

```diff
@@ -0,0 +1,31 @@
+package sbtcross
+
+import sbt._
+import StringUtilities.nonEmpty
+
+import scala.language.implicitConversions
+
+object SbtCrossPlugin extends AutoPlugin {
+  val autoImport = AutoImport
+  object AutoImport extends JvmCross with CrossKeys {
+    final implicit def toScalaJSGroupID(groupID: String) = null
```

Comment:
This must have an explicit result type.

Also, rather than returning `null`, it should at the very least throw an exception.